classes/Location.php

```diff
@@ -158,12 +175,12 @@ public function collectionSchedule()
 
     public function isOpened()
     {
-        return $this->workingSchedule($this->orderType())->isOpen();
+        return $this->getOrderType()->getSchedule()->isOpen();
```

Comment:
closeTime(), opentime(), workingStatus() also need the same change applied

Reply:
I didn't change those methods so we can return `opening` schedules as well for reservation or whatever since we are not registering an `opening` order type.